chapters/synchronous.tex

```diff
@@ -1083,6 +1092,13 @@ \subsection{Clocked Discrete-Time and Clocked Discretized Continuous-Time Partit
 Such a partition has to be solved with a \emph{solver method} of \cref{solver-methods}.
 When \lstinline!previous(x)! is used on a continuous-time state variable \lstinline!x!, then \lstinline!previous(x)! uses the start value of \lstinline!x! as value for the first clock tick.
 
+The use of the operator sample from \cref{event-related-operators-with-function-syntax} in a clocked partition is problematic.
+Diagnostics are appropriate especially if it is intended to generate sampling faster than the clocks, and otherwise the sampling should ideally be adjusted to the clock ticks.
+\begin{nonnormative}
```

Comment:
Empty lines on both sides of `nonnormative` increase readability of the source:
```suggestion

\begin{nonnormative}
```

Reply:
Neither resolved nor commented.  Marked as resolved by mistake?

Reply:
No, I believe it was merged, **and then** that entire sample-part was cut out to a separate PR #3099 - where you can see the empty lines.

Reply:
OK, resolving.